src/inverseRobot.js

```diff
@@ -23,8 +23,53 @@
  *
  * @return {object}
  */
+
 function inverseRobot(robot) {
-  // write code here
+  if (!robot) {
+    return {};
+  }
+
+  const keys = Object.keys(robot).reverse();
+  const values = Object.values(robot).reverse();
```

Comment:
why do you reverse these?

Reply:
resulted array with items was reversed,  was thinking that tests won`t pass,
but any way - I found a shorter solution